### DISCUSSION TITLE
Refresh GUI and Verbose

### DISCUSSION THREAD

**Discussion by Greigh** [wstyres/Zebra#1091] [Ideas]
**Is your feature request related to a problem? Please describe.**
I get frustrated when it is refreshing sources and I can’t figure out which source is taking so long to refresh so it would be nice to have some kind of option for verbose and/or a GUI interface when refreshing sources. 

**Describe the solution you'd like**
A GUI and/or Verbose option in settings. 

**Describe alternatives you've considered**
Maybe show the number of repos that still need to be refreshed somewhere? This still wouldn’t tell me which one it is but it could be helpful info if you aren’t considering the above.

**Comment by jakeva** [comment]
I'm not sure if I completely understand your issue. However, there already is an icon next to each source that is refreshing.

**Comment by Greigh** [comment]
Yes. But it is frustrating scrolling through all of my repos to find which repo is having issues refreshing.

**Comment by wstyres** [comment]
The simplest implementation that I can see would be just to show the refresh popup every time if the option is on. Would that be enough or did you have something else in mind?

**Comment by Greigh** [comment]
Well I was looking for something that would show the progress of how many repos need to be refreshed.
